scripts/cartographer_offline.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#
+#   Cartographer Offline Script
+#   Must be run in the ros.package directory
+#
+scriptdir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+logpath=$scriptdir/../ros_ws/src/navigation_stack/car_cartographer/files/cartographer.log
+
+source $scriptdir/../ros_ws/devel/setup.bash
+
+echo -n "Create .urdf file... " | tee $logpath
+echo "" >> $logpath
+xacro -o $scriptdir/../ros_ws/src/navigation_stack/car_cartographer/files/racer.urdf --inorder $scriptdir/../ros_ws/src/simulation/racer_description/urdf/racer.xacro use_gpu:=true visualize_lidar:=true 2>&1 | sed -r "s/([[:cntrl:]]\[[0-9]{1,3}m)|(..;?)|//g" >> $logpath
+echo "Done." | tee -a $logpath
+
+if [ $ROS_DISTRO == "kinetic" ] ; then
+    echo -n "Create map... " | tee -a $logpath
+    echo "" >> $logpath
+    roslaunch car_cartographer cartographer_offline_fast.launch bag_filenames:=$1 ros_version:=$ROS_DISTRO 2>&1 | sed -r "s/([[:cntrl:]]\[[0-9]{1,3}m)|(..;?)|//g" >> $logpath
```

Comment:
Consider creating a variable for the `"s/([[:cntrl:]]\[[0-9]{1,3}m)|(..;?)|//g"` string (also in the other `.sh` file)

Reply:
Done.